README.md

```diff
@@ -51,12 +57,30 @@ This is just JS, so feel free to put Real Code in there.
 
 ## 3. How to invoke webpack
 
+You should install webpack globally:
```

Comment:
Why would you do this?

I don't recommend it. This should be specified as a dev dependency in `package.json`. Your builds and running the hot reload server will be sensitive what version of webpack you're using.